src/components/widgets/Number.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react'
+import Header from './Header'
+import "../../styles/widget/number.css"
+import "../../styles/widget/widget.css"
+
+class Number extends Component {
+  render() {
+    const { title, x, y, statistic, goal } = this.props
+    const positionStyle = { top: x, left: y }
+
+    const progressRatio = ( statistic / goal ) * 100
+    const progressRounded = +progressRatio.toFixed( 2 )
+    const progressPercentage = `${progressRounded}%`
+    const isComplete = statistic / goal >= 1
+
+    return (
+      <article>
+        <div className="widget widget-width-1 widget-height-1"
+          style={ positionStyle }>
+          <Header title={ title }/>
+          <div className="widget-body stat-wrapper">
+            <div className="statistic">{ statistic }</div>
+            <div className="goal-progress-wrapper">
+              <div className={ isComplete
```

Comment:
This is very jarring to read - if it's less than 80 characters, it can be on a single line.  Also, the `progress` class is repeated:
```jsx
<div className=`progress${isComplete ? : ' progress-complete' : ''}`>
```
I also note that you have this `isComplete` test happening over and over...  We could DRY out this code by creating selectors that leverage a single instance of a `completed` class:
```css
.completed progress {
  // Whatever the progress-complete rules are
}

.completed goal {
  // Whatever the goal-complete rules are
}

// etc.
```
Now, we can perform the test once, and place the completed class around these other classes:
```html
<div className=`goal-progress-wrapper${isComplete ? ' completed' : ''}`>
  <div className="progress">{ progressPercentage }</div>
  <div className="goal">{ goal }</div>
  <div className="progress-bar">
    <span style={{ width: `${Math.ceil( progressPercentage )}%` }}></span>
  </div>
</div>
```